src/app/work-item/work-item-list/work-item-detail/work-item-link/work-item-link.component.html

```diff
@@ -99,8 +99,8 @@ <h2 class="col-md-12 col-sm-12 col-xs-12 dib">Work Item Links</h2>
             <div class="link-list-wrap" *ngIf="workItem.relationalData.linkDicts">
                 <h4 class="link-list-header">Links</h4>
                 <div *ngFor='let linkDict of workItem.relationalData.linkDicts'>
-                    <h5><strong>{{linkDict.linkName}}</strong></h5>
-                    <ul>
+                    <h5 class="col-md-12 col-sm-12 col-xs-12 no-padding">{{linkDict.linkName}}</h5>
```

Comment:
While we can set this, it is really up to the @almighty/uxd team to decide the font weight for each item.